website/blog/2024-07-11-what-is-a-data-lakehouse.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "What is a Data Lakehouse & How does it Work?"
+excerpt: "Explains the concept of the lakehouse architecture"
+author: Dipankar Mazumdar
+category: blog
+image: /assets/images/blog/dlh_1200.png
+tags:
+- data lakehouse
+- Apache Hudi
+- Apache Iceberg
+- Delta Lake
+- Open Architecture
+---
+
+A data lakehouse is a hybrid data architecture that combines the best attributes of data warehouses and data lakes to address their respective limitations. This innovative approach to data management brings the transactional capabilities of data warehouses to cloud-based data lakes, offering scalability at lower costs. 
+
+![/assets/images/blog/dlh_new.png](/assets/images/blog/dlh_new.png)
```

Comment:
Do we need to add this image. The images in the PR are named differently than here.

Reply:
fixed!